sdk/storage/azure-storage-files-datalake/src/datalake_file_client.cpp

```diff
@@ -230,7 +240,12 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     protocolLayerOptions.RetainUncommittedData = options.RetainUncommittedData;
     protocolLayerOptions.Close = options.Close;
     protocolLayerOptions.ContentLength = 0;
-    protocolLayerOptions.ContentMd5 = options.ContentMd5;
+    if (options.ContentHash.HasValue()
+        && options.ContentHash.GetValue().Algorithm == HashAlgorithm::Crc64)
+    {
+      abort();
+    }
+    protocolLayerOptions.ContentMd5 = options.ContentHash;
```

Comment:
Again, what is in `options.ContentHash`? Transactional or part of `HttpHeaders`?

Reply:
It will be moved to HttpHeaders.